### PR TITLE
Compact image modal layout

### PR DESCRIPTION
**Pull request by ciar4n**
Pull Request for Issue # .

### Summary of Changes
Compacts the media field modal layout and increases modal size. Removes the double scroll bar on larger screens. Currently, there are 3 options to close. This PR removes the close button in the footer (admin).

### Testing Instructions
Apply PR and open administration media modal (media field -> select).

### Before
image

### After
image

### CODE REVIEW

File: administrator/components/com_media/views/images/tmpl/default.php [hunk 1]
```diff
@@ -65,28 +65,30 @@
...
 				</div>
-				<div class="pull-right">
-					<button class="btn btn-success button-save-selected" type="button" <?php if (!empty($onClick)) :
-					// This is for Mootools compatibility ?>onclick="<?php echo $onClick; ?>"<?php endif; ?> data-dismiss="modal"><?php echo JText::_('COM_MEDIA_INSERT'); ?></button>
-					<button class="btn button-cancel" type="button" onclick="window.parent.jQuery('.modal.in').modal('hide');<?php if (!empty($onClick)) :
-						// This is for Mootools compatibility ?>parent.jModalClose();<?php endif ?>" data-dismiss="modal"><?php echo JText::_('JCANCEL'); ?></button>
+				<div class="span4 control-group">
+					<div class="pull-right">
+						<button class="btn btn-success button-save-selected" type="button" <?php if (!empty($onClick)) :
+						// This is for Mootools compatibility ?>onclick="<?php echo $onClick; ?>"<?php endif; ?> data-dismiss="modal"><?php echo JText::_('COM_MEDIA_INSERT'); ?></button>
```
Comment (Quy) [inline administrator/components/com_media/views/images/tmpl/default.php]: Indent to match below?

**Comment by Quy** [comment]
Hopefully, you can come up with a fix for this related issue #17551.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on ed85e70d477c1509019d7a21673f871f538bebdd<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22475">issues.joomla.org/tracker/joomla-cms/22475</a>.</sub>

**Comment by ciar4n** [comment]
> Hopefully, you can come up with a fix for this related issue #17551.

I will have a look. Better suited for a separate PR as it will likely touch all tiny mce modals.

**Comment by Quy** [comment]
Toolbar is no longer pinned with this PR.

**Comment by ciar4n** [comment]
Not sure if I understand. Would you have a screenshot?

**Comment by Quy** [comment]
With this PR, go to `Content > Articles`, scroll down and the toolbar goes off screen and no longer is pinned.

toolbar

**Comment by ciar4n** [comment]
Strangely I am unable to replicate this issue. Toolbar is sticking in both Firefox and Chrome.

**Comment by Quy** [comment]
Using your branch, it is not an issue. However, with beta4 and patchtester, then it is reproducible.

**Comment by mbabker** [comment]
An issue because of patchtester doesn't mean the patch has issues.  It could very well be there is another change after beta 4 that patchtester isn't applying which is causing what you're seeing.

It is literally impossible to make patchtester behave the same as if you're running `git apply PR.diff` because the CMS environment does not have the git environment data necessary to generate the appropriate diffs.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on ed85e70d477c1509019d7a21673f871f538bebdd<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22475">issues.joomla.org/tracker/joomla-cms/22475</a>.</sub>

**Comment by brianteeman** [comment]
It's better but I still have a small scroll (unlike your screenshot)

Tested on Chrome 70 Windows
image

**Comment by ciar4n** [comment]
@brianteeman Should be ok now.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on c2d7df218ff42fbaa3fdcb47702ef26d7d7a04fb<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22475">issues.joomla.org/tracker/joomla-cms/22475</a>.</sub>

**Comment by brianteeman** [comment]
Tested again with 
Firefox 64.0.2 (64-bit) Windows 
Version 71.0.3578.98 (Official Build) (64-bit) Windows

I still have the same results as in https://github.com/joomla/joomla-cms/pull/22475#issuecomment-434515499

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on 2f7270c6b081432be217c339a7cb080177b94e3b<br />
### System information
- 3.9, latest nightly build
- Template: Protostar
- macOS High Sierra, 10.13.6
- Firefox 66 (64-bit)
#### CloudAccess.net
- PHP 7.1.27
- MySQLi 5.7.24-cll-lve
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22475">issues.joomla.org/tracker/joomla-cms/22475</a>.</sub>

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on 2f7270c6b081432be217c339a7cb080177b94e3b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22475">issues.joomla.org/tracker/joomla-cms/22475</a>.</sub>

**Comment by ghost** [comment]
Status "Ready To Commit".

**Comment by roland-d** [comment]
I don't think this can be RTC as @brianteeman is still seeing an issue

**Comment by ghost** [comment]
@roland-d thanks, had only looked at number of tests. 

Status back on "Pending".

**Comment by Quy** [comment]
@brianteeman Can you please test again?

Tested fine on Windows 10:
Firefox 67.0.2
Chrome 75.0.3770.80

**Comment by Quy** [comment]
@roland-d We have 3 successful tests. I can't reproduce the issue Brian is having. Can we please merge it anyway? It is better than before and deal with the issue Brian has in a later PR if more people report it.

**Comment by roland-d** [comment]
@HLeithner As release leader, I will let you decide.

**Comment by HLeithner** [comment]
It works for me fine in ff and vivaldi. I have have no idea why brian has this problem but I think its better to merge it then having the current state.

thx

**Comment by ciar4n** [comment]
Thanks all for the tests

**Pull request merged**